Schyntax/Schedule.cs

```diff
@@ -342,6 +370,21 @@ private static int CompareMonthAndDay(int monthA, int dayA, int monthB, int dayB
             return monthA > monthB ? 1 : -1;
         }
 
+        private static bool InDayOfYearRange(IrIntegerRange range, int year, int month, int dayOfYear)
```

Comment:
True. Btw, one of the other methods that was there also had one of these IIRC. Will refactor.